strategy/exception/default_exception_formatting_strategy.go

```diff
@@ -115,7 +116,8 @@ func (dEFS *DefaultFormattingStrategy) Panicf(formatString string, args ...inter
 // ExceptionFromError takes an error and returns value of Exception
 func (dEFS *DefaultFormattingStrategy) ExceptionFromError(err error) Exception {
 	var isRemote bool
-	if reqErr, ok := err.(awserr.RequestFailure); ok {
+	var reqErr awserr.RequestFailure
```

Comment:
Also, I think in general it would be a good idea to add some test cases around this to confirm the behavior.

Reply:
Sure, I've added some basic cases